open-life-science-mentoring/README.md

```diff
@@ -4,6 +4,11 @@
 
 ## The Turing Way collaborates with OLS: *OLS-2 for Turing*
 
+## OLS-3 for Turing projects
+
+Congratulations to the following 6 projects that will join the second cohort (OLS-3) under the collaboration *OLS-3 for Turing*.
```

Comment:
Second or third cohort? You refer to second cohort but list OLS-3 here :)

Reply:
Thanks @Arielle-Bennett. This was a test PR for @alihumayun who will update it when he is back. :)